source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts

```diff
@@ -55,6 +55,7 @@ export default {
     pictureTitle: 'Profile Picture',
     general: 'General',
     updated: 'User updated successfully!',
+    updatePicture: 'Update your picture at Gravatar.com',
```

Comment:
Is it guaranteed that google-auth2 get picture from Gravatar.com?

Reply:
this isn't related to google-auth2 --- but yeah, I guess maybe the picture is gotten from other locations. no idea atm.